lib/better_errors/error_page.rb

```diff
@@ -1,6 +1,7 @@
 require "cgi"
 require "json"
 require "securerandom"
+require "objspace"
```

Comment:
[JRuby has ObjectSpace disabled by default and recommends leaving it off for performance reasons](https://github.com/jruby/jruby/wiki/PerformanceTuning#dont-enable-objectspace), so we need to make sure that if the `require` fails, we rescue from that exception and disable any use of `ObjectSpace` if it's not defined.